docs/runtime-contract.md

```diff
@@ -484,16 +484,16 @@ From the OCI spec:
 > article in the kernel documentation has more information about mount
 > propagation.
 
-This option should only be set by the operator or platform provider, and MUST
-NOT be configurable by the developer. As mount propagation may be part of the
-platform security hardening, operators MAY tune this over time as the threat
+This option SHOULD only be set by the operator or platform provider, and MUST
+NOT be configurable by the developer. As mount propagation might be part of the
+platform security hardening, operators might tune this over time as the threat
```

Comment:
If 474 is MAY, this should be MAY as well.

Reply:
+1 I would make both of these MAY

Reply:
Are we really adding constraints here or just stating how things are? It didn't come across to me like we were being normative. For example, the phrase "person X might do something over time" comes across to me more like "guidance" or explaining what's going on in the system rather than saying "make sure person X is given the option of doing ....".  If we really are saying "make sure the operation is given the option of doing this" then I think we need to word it more strongly so it's not so ad hoc-ish.

Reply:
I think we want to explicitly word it so that these tools are available for operators.

Would "Operators MAY disable or force this setting to a particular value" be better?

Reply:
see if the new text is ok